tea.rb

```diff
@@ -0,0 +1,46 @@
+require "formula"
+
+class Gitea < Formula
+  homepage "https://github.com/go-gitea/tea"
+  head "https://github.com/go-gitea/tea.git"
+
+  stable do
+    version "1.0.0"
```

Comment:
TODO: tag a version for tea

Reply:
Since `tea` is too early I would like name it version `0.1.0`.